src/components/checkbox/index.tsx

```diff
@@ -139,6 +147,7 @@ class Checkbox extends Component<CheckboxProps, CheckboxState> {
       ]
     };
   }
+  validationState = false;
```

Comment:
Not sure why `validationState` is needed, I think this check can cause issues with `Form` (we might need to add an `isValid` function for it, which might get buggy from this).

Reply:
`validationState` is a private flag that makes sure the validation check will only be performed after the user invoked the validate() method. The Form should only perform the validate().

Reply:
I don't see the value that it adds, is there an example it is useful in\for?

Reply:
It looks to me like it is serving no purpose, and if this is the case we should remove it.